### PR TITLE
dpdk-pipeline-cli file added as templete to add/remove/modify the rul…

### PR DESCRIPTION
**Pull request by MD1614-TUD**
…es on the fly

### CODE REVIEW

File: control-plane-configs/cp1.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 - ip_pool4_net: 192.168.4.0/24
 - telnet_port: 2000
 - tcp_port: 2001
-- hostname_dataplane: "data-plane-01.labor2.bisdn.de"
```
Comment (jklare) [inline control-plane-configs/cp1.yml]: these are just examples and should not be changed

File: control-plane-configs/cp2.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@
 - ip_pool4_net: 192.168.14.0/24
 - telnet_port: 4000
 - tcp_port: 4001
-- hostname_dataplane: "data-plane-01.labor2.bisdn.de"
```
Comment (jklare) [inline control-plane-configs/cp2.yml]: these are just examples and should not be changed

File: templates/accel-ppp.j2 [hunk 3]
```diff
@@ -299,8 +299,8 @@ pubchan=accel-ppp
 # select the event types to emit a message via redis
 #
 ev_ses_starting=yes
-#ev_ses_finishing=yes
```
Comment (jklare) [inline templates/accel-ppp.j2:302]: this should be mentioned and explained in the commit message

**Review by jklare — COMMENTED** [review]
lgtm, some comments

File: templates/accel-ppp.j2 (hunk 3 above)
Comment (MD1614-TUD) [reply templates/accel-ppp.j2:302]: Ok.

**Comment by MD1614-TUD** [comment]
The commits have been added to master. So deleting the branch.

**Pull request merged**